hoot-core/src/main/cpp/hoot/core/ops/BuildingOutlineUpdateOp.cpp

```diff
@@ -124,22 +125,14 @@ class NodeReplaceVisitor : public ConstElementVisitor
   OsmMap& _map;
 };
 
-BuildingOutlineUpdateOp::BuildingOutlineUpdateOp() :
-_removeBuildingRelations(false)
+BuildingOutlineUpdateOp::BuildingOutlineUpdateOp()
 {
```

Comment:
Removing building relations here was a hack, which I didn't realize was a hack until finishing this PR.